[2024-12-19T05:43:30.912+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T05:43:30.918+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2024-02-28T00:00:00+00:00 [queued]>
[2024-12-19T05:43:30.922+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2024-02-28T00:00:00+00:00 [queued]>
[2024-12-19T05:43:30.923+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T05:43:30.930+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2024-02-28 00:00:00+00:00
[2024-12-19T05:43:30.933+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4109) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T05:43:30.933+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2024-02-28T00:00:00+00:00', '--job-id', '948', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpzuz58i59']
[2024-12-19T05:43:30.935+0000] {standard_task_runner.py:72} INFO - Started process 4116 to run task
[2024-12-19T05:43:30.935+0000] {standard_task_runner.py:105} INFO - Job 948: Subtask first_task
[2024-12-19T05:43:30.956+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2024-02-28T00:00:00+00:00 [running]> on host 13c961a72918
[2024-12-19T05:43:30.995+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-28T00:00:00+00:00'
[2024-12-19T05:43:30.996+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T05:43:31.004+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T05:43:31.004+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T05:43:31.008+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T05:43:31.009+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T05:43:31.009+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T05:43:31.022+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T05:43:31.023+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2024-02-28T00:00:00+00:00, execution_date=20240228T000000, start_date=20241219T054330, end_date=20241219T054331
[2024-12-19T05:43:31.033+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T05:43:31.038+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
